release.yaml

```diff
@@ -543,6 +543,116 @@ promoteToPublic:
           Promoted release is **publicly available** through a git tag at [\`{{version}}\`](https://github.com/sourcegraph/deploy-sourcegraph-helm/tree/{{version}}).
           EOF
 
+      - name: "update main with latest version"
+        cmd: |
+          set -eu
+          branch="promote/release-{{version}}-update-main"
+          echo "Checking out origin/main"
+          git fetch origin main
+          git switch main
+          echo "Creating branch origin/${branch}"
+          git switch -c "${branch}"
```

Comment:
The earlier step is just assigning the name right? The `git switch main` part is also confusing to me though. In all these deploy repos we're now cutting releases from a release branch, so this likely needs to be whatever the base branch `sg release <>`

Otherwise this just looks like making sure things are up to date with main 🤔

Reply:
So the idea here is to update `main` with the latest version of Sourcegraph. So to do that we:

- checkout `main`
- create a new PR off `main` with the updated version and tags we need
  - create a new branch
  - make the changes
  - commit the changes
  - open the PR

We don't really need to do anything with the other release PRs. They are similar, but not really related. The idea here is to keep them separated and intentionally create a PR for the changes we want made to `main`.

Unless _I'm_ missing something here 😆